tests/Microsoft.Identity.Test.Unit.net45/PublicApiTests/ConfidentialClientApplicationTests.cs

```diff
@@ -783,6 +783,26 @@ public async Task AuthorizationCodeRequestTestAsync()
                 }).ConfigureAwait(false);
         }
 
+        [TestMethod]
+        [TestCategory("ConfidentialClientApplicationTests")]
+        public async Task AcquireTokenByRefreshTokenTestAsync()
+        {
+         await RunWithMockHttpAsync(
+                async (httpManager, serviceBundle, receiver) =>
+                {
+                    httpManager.AddInstanceDiscoveryMockHandler();
+                    httpManager.AddMockHandlerForTenantEndpointDiscovery(MsalTestConstants.AuthorityCommonTenant);
+                    httpManager.AddSuccessTokenResponseMockHandlerForPost();
+
+                    TokenCache userCahce = new TokenCache();
+                    ClientCredential cc = new ClientCredential(MsalTestConstants.ClientSecret);
+                    ConfidentialClientApplication app = new ConfidentialClientApplication(serviceBundle, MsalTestConstants.ClientId, MsalTestConstants.AuthorityCommonTenant, MsalTestConstants.RedirectUri, cc, userCahce, new TokenCache());
+                    await (app as IByRefreshToken).AcquireTokenByRefreshTokenAsync("SomeRefreshToken").ConfigureAwait(false);
```

Comment:
Are we expecting our developers to have to do this "as" casting to be able to access this explicitly implemented method?  Hiding it this way doesn't seem great.